client/src/routes/automation/esidom_generator.ts

```diff
@@ -86,7 +86,8 @@ interface BlocksGenerator {
             json.action = JSON.parse(actions);
         }
 
-        json.dropdown_mode = dropdown_mode;
+        // json.dropdown_mode = dropdown_mode;
+        json.dropdown_mode = 'single';
```

Comment:
A comment may be useful to understand why it's hard coded ?

Reply:
Yes sir